tools/dimspy/process_scans.xml

```diff
@@ -195,6 +193,31 @@
                 <element name="batch04_QC18_rep03_282" file="batch04_QC18_rep03_282.txt" ftype="tsv"/>
             </output_collection>
         </test>
+        <test>
+            <param name="input|format" value="data_collection" />
```

Comment:
I think the better syntax is this here:

https://github.com/galaxyproject/tools-iuc/blob/57493fdc88e6128bed127993464f2a6480e2774d/tools/enasearch/enasearch_search_data.xml#L82